### PR TITLE
Update requirements for noble

### PR DESCRIPTION
**Pull request by aieri**
This commit unpins requirements, allowing for unit tests to pass on
Ubuntu noble (Python 3.12)

Partial-fix: #16

### CODE REVIEW

**Comment by aieri** [comment]
this PR supersedes #29

**Review by Pjack — APPROVED** [review]

**Review by samuelallan72 — DISMISSED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-ops~=1.3.0
+charmhelpers @ git+https://github.com/juju/charm-helpers@master
 click
-fabric~=2.6.0
+fabric
+ops
 python-openstackclient
-PyYAML~=6.0
\ No newline at end of file
+PyYAML
```
Comment (samuelallan72) [inline requirements.txt:6]: @aieri does this affect the charm build for bionic or focal though?

Comment (aieri) [reply requirements.txt:6]: possibly, but we could drop both

**Comment by aieri** [comment]
superseded by #36

**Pull request closed without merging**